### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by davidsbond**
Update badges to markdown equivalents and use GitHub actions badge instead of prow.
Update references to "preflight" to "Jetstack Secure" where appropriate.

Signed-off-by: David Bond <davidsbond93@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,25 +1,21 @@
...
-
-Preflight is a tool to automatically perform Kubernetes cluster configuration
+# Jetstack Secure Agent
+
+[![release-master](https://github.com/jetstack/preflight/actions/workflows/release-master.yml/badge.svg)](https://github.com/jetstack/preflight/actions/workflows/release-master.yml)
+[![Go Reference](https://pkg.go.dev/badge/github.com/jetstack/preflight.svg)](https://pkg.go.dev/github.com/jetstack/preflight)
+[![Go Report Card](https://goreportcard.com/badge/github.com/jetstack/preflight)](https://goreportcard.com/report/github.com/jetstack/preflight)
+
+
+The Jetstack Secure Agent is a tool to automatically perform Kubernetes cluster configuration
```
Comment (tfadeyi) [inline README.md:8]:
```suggestion
The Jetstack Secure is a tool to automatically perform Kubernetes cluster configuration
```

Comment (tfadeyi) [reply README.md:8]: We could mention cert inventory together with the opa checks

**Review by tfadeyi — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -29,7 +25,7 @@ markdown-toc-refresh-toc -->
 
 ## Project Background
 
-Preflight was originally designed to automate Jetstack's production readiness
+Jetstack Secure was originally designed to automate Jetstack's production readiness
 assessments.
 These are consulting sessions in which a Jetstack engineer inspects a customer's
 cluster to suggest improvements and identify configuration issues.
```
Comment (j-fuentes) [inline README.md:31]: This copy reflects the original idea of Preflight, but Jetstack Secure encompasses more things. I would use similar terms to what we say in: https://www.jetstack.io/jetstack-secure/.

File: README.md [hunk 3]
```diff
@@ -1,25 +1,21 @@
...
-<img alt="Build Status" src="https://prow.build-infra.jetstack.net/badge.svg?jobs=post-preflight-release-canary">
-</a>
-<a href="https://godoc.org/github.com/jetstack/preflight"><img src="https://godoc.org/github.com/jetstack/preflight?status.svg"></a>
-<a href="https://goreportcard.com/report/github.com/jetstack/preflight"><img alt="Go Report Card" src="https://goreportcard.com/badge/github.com/jetstack/preflight" /></a>
-</p>
-
-# Jetstack Preflight
-
-Preflight is a tool to automatically perform Kubernetes cluster configuration
+# Jetstack Secure Agent
```
Comment (j-fuentes) [inline README.md:1]: The name of the repo is still preflight. So I would add a note explaining that preflight is now part of Jetstack Secure.

**Comment by charlieegan3** [comment]
replaced with #236

**Pull request closed without merging**